include/libtorrent/bandwidth_limit.hpp

```diff
@@ -47,16 +43,17 @@ namespace libtorrent {
 // member of peer_connection
 struct TORRENT_EXTRA_EXPORT bandwidth_channel
 {
-	static const int inf = boost::integer_traits<int>::const_max;
+	static constexpr int inf = std::numeric_limits<std::int32_t>::max();
```

Comment:
it seems weird that the type here is ``int``, since the numeric limit is from ``int32_t``